patch/llpcConfigBuilderBase.h

```diff
@@ -59,7 +59,11 @@ class ConfigBuilderBase
     void SetIndirectTableEntry(uint32_t value);
     void SetStreamOutTableEntry(uint32_t value);
 #endif
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
```

Comment:
I don't think you need #if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= 35
before version 35, ShaderHash is uint64_t